data/brands/shop/chemist.json

```diff
@@ -10,6 +10,18 @@
     }
   },
   "items": [
+     {
+      "displayName": "Caddy's",
+      "locationSet": {"include": ["it"]},
+      "matchTags": ["shop/conveneience", "shop/houseware", "shop/cosmetics"],
```

Comment:
Hello @alessandrocarmeli,
thanks for your submission. Could you please correct the spelling on the matchTags (shop/conve**nie**nce).
Also, are you sure that the shop/houseware is appropriate? I'm not saying it isn't, I just wouldn't have expected it for a place that's primarily tagged as a chemist.
Thanks,

Reply:
I'm sure